app/src/main/java/ciserver/ContinuousIntegrationServer.java

```diff
@@ -47,6 +52,13 @@ private void handlePushEvent(PushEvent event) {
         System.out.println("Commit status update sent successfully.");
     }
 
+    /**
+     * See
+     * <a
+     * href=https://www.eclipse.org/jetty/javadoc/jetty-9/org/eclipse/jetty/server/handler/AbstractHandler.html#handle(java.lang.String,org.eclipse.jetty.server.Request,javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpServletResponse)>documentation
+     * of the parent interface</a>
+     * {@inheritDoc}
```

Comment:
Unsure if just linking out like this is good enough

Reply:
I feel like i noticed you having an idea with `@see` somewhere - that sounds like a good addition. I think this will most likely be "OK" as-is, but it could definitely use some clarification.

Reply:
Yes, I originally thought `@see` (or `@link`) would be able to generate a link to the external documentation or take the URL as an argument, but the `JavaDoc` tool seems unable to find the documentation for the jetty library on its own (and `@see`/`@link` don't seem to work with URLs). I think a plain HTML link, as it is now, is the only way to actually link to the documentation of the parent interface.

The closest I got while using `@link` was this: `{@link org.eclipse.jetty.server.handler.AbstractHandler#handle(String, Request, HttpServletRequest, HttpServletResponse)}` but that does not generate a clickable link (the same tactic DOES work with a `java.net....` class, if you look at `PipelineUpdateRequest.java`)

The alternative/a possible addition would be to basically just copy and paste from the jetty documentation into our javadoc.

Reply:
I see. In that case this is most likely enough. Could we add a short description, though ? Something like `handles an HttpServletRequest. See <a href= […]`.

Reply:
Good idea, will do